zerver/tests/test_email_log.py

```diff
@@ -32,8 +32,7 @@ def test_forward_address_details(self) -> None:
 
         self.assertEqual(get_forward_address(), forward_address)
 
-        with self.settings(EMAIL_BACKEND='zproject.email_backends.EmailLogBackEnd'), \
-                mock.patch('logging.info', return_value=None):
+        with self.settings(EMAIL_BACKEND='zproject.email_backends.EmailLogBackEnd'):
             with mock.patch('zproject.email_backends.EmailLogBackEnd.send_email_smtp'):
```

Comment:
I assume this is case where the code didn't actually log anything?

Reply:
Yeah, there was no logging,at places where I have removed patch for logging.